src/Common/src/System/Security/Cryptography/Asn1/AsnXml.targets

```diff
@@ -0,0 +1,43 @@
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <AsnXml />
+  </PropertyGroup>
+  <ItemGroup>
+    <None Include="$(MSBuildThisFileDirectory)asn.xsd">
+      <Link>Common\System\Security\Cryptography\Asn1\asn.xsd</Link>
+    </None>
+  </ItemGroup>
+
+  <!-- MSBuild for .NET Core doesn't support XslTransform at this time -->
+  <Target Name="CompileAsn" BeforeTargets="CoreCompile"
+    Condition="'$(RunningOnCore)' != 'true'"
+    Inputs="@(AsnXml);$(MSBuildThisFileDirectory)asn.xslt"
+    Outputs="%(Identity).cs">
```

Comment:
What does `%(Identity)` set to at this point? Looks like you are trying to batch on something but it isn't clear what .

Reply:
Never mind I found AsnXml being set in the project.